### DISCUSSION TITLE
Can you do routing in combination with parallel edges?

### DISCUSSION THREAD

**Discussion by tomzeppenfeldt** [antvis/X6#1209] [Q&A] [answered]
We have been looking at X6 / G6 and it looks very promising. We have a use case that requires a combination of manhattan or metro routing in combination with parallel edges. The edges have different labels and may have opposite directions. The reason why we need parallel edges is that users should be able to do CRUD on individual edges.

Is this something G6 / X6 supports? We were not able to read the full documentation (because of the language), so perhaps we missed it among the plethora of examples.

**Comment by NewByVector** [comment, marked as answer]
You can refer to this [demo](https://github.com/antvis/X6/blob/master/examples/x6-example-features/src/pages/connector/er-rounded.tsx)
